doc/dev/credscan_process.md

```diff
@@ -0,0 +1,80 @@
+# Guide for monitoring CredScan checks
+
+This guide describes how package owners can monitor their package's Credential Scanner (CredScan) status and correct
+any warnings.
+
+General information about CredScan can be found in the overview documentation at [aka.ms/credscan][credscan_doc]. The
+Azure SDK's motivation and methodology for running CredScan is documented [here][devops_doc].
+
+## Table of Contents
+- [Check CredScan status](#check-credscan-status)
+- [Correct active warnings](#correct-active-warnings)
+  - [True positives](#true-positives)
+  - [False positives](#false-positives)
+- [Correct baselined warnings](#correct-baselined-warnings)
+
+## Check CredScan status
+
+CredScan is run each night over the entire `azure-sdk-for-python` repository as part of the
```

Comment:
We are considering to add the CredScan check inside of PR validation in .NET repo and once we get that figured out we could do that similarly in python repo if you guys feel that would be helpful for you guys.

Reply:
That would definitely be great to implement once that's ready! I synced with Sima about the PR validation work and think it'll be a great solution moving forward. This guide will hopefully help Python cut through our current backlog of CredScan warnings and get us in a position to use PR validation.